addons/resource.language.en_gb/resources/strings.po

```diff
@@ -11636,13 +11636,13 @@ msgstr ""
 
 #: system/settings/settings.xml
 msgctxt "#21472"
-msgid "Include All Seasons and Specials"
+msgid "Include all seasons and specials"
 msgstr ""
 
 #. Description of setting "Videos -> Library -> Include All Seasons and Specials" with label #21472
 #: system/settings/settings.xml
 msgctxt "#21473"
-msgid "Whether or not to consider All Seasons and Special items in the unwatched item selection."
+msgid "Whether or not to consider all seasons and special items in the unwatched item selection."
```

Comment:
I'm not sure about the "All Seasons" -> "all seasons" changes in this area, because it's no longer obvious that these are referenes to other sections/labels. So maybe: